content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -626,8 +631,8 @@ case.
 
 With the Server Side Apply feature enabled, the `PATCH` endpoint accepts the
 additional `application/apply-patch+yaml` content type. Users of Server Side
-Apply can send partially specified objects to this endpoint. An applied config
-should always include every field that the applier has an opinion about.
+Apply can send partially specified objects as yaml or json to this endpoint.
+An applied config should always include every field that the applier has an opinion about.
```

Comment:
Can you reword? A reader might think that the applier is the API server; as a piece of code, API servers don't have opinions.